app/profile-page.html

```diff
@@ -0,0 +1,143 @@
+               <img src="img/logo.png" alt="Chata logo">
+               <h1>Your Profile</h1>
+               <form method="POST" ng-submit="submit()">
+                  <p>Profile Picture (Optional)</p>
+                  <div class="form-group">
+                     <img id="uploadedPicture" src="img/avatar.png" />
+                     <div id="hiddenFileUploader"><input id="upfile" type="file" value="upload" onchange="readURL(this)" /></div>
+                  </div>
+                  <br>
+                  <div class="form-group">
+                     <label class="sr-only" for="username">Username</label>
+                     <input type="text" class="form-control" id="username" placeholder="Username" ng-model="usr_username" >
+                  </div>
+                  <div class="form-group">
+                     <label class="sr-only" for="loginEmail">Email</label>
+                     <input type="email" class="form-control" id="loginEmail" placeholder="Email" ng-model="usr_email" >
+                  </div>
+                  <p class="text-danger" ng-if="error.show"><small>{{error.message}}</small></p>
+                  <fieldset>
+                     <ng-form>
+                        <div class="form-group">
+                           <label class="sr-only" for="loginPassword">Current Password</label>
+                           <input type="password" ng-pattern="/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/" class="form-control" id="loginPassword" placeholder="Current Password" ng-model="usr_password" >
+                           <!--<div ng-show="form.$submitted || form.pass.$touched">
+                              <span ng-show="form.pass.$error.required">Enter your current password.</span>
+                              <span ng-show="form.pass.$error.pattern">Password must have 8 character, 1 letter and 1 number.</span>
+                           </div> -->
+                        </div>
+                        <div class="form-group">
+                           <label class="sr-only" for="newPassword">New Password</label>
+                           <input type="password" ng-pattern="/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/" class="form-control" id="newPassword" placeholder="New Password" ng-model="usr_newPassword" >
+                           <!-- <div ng-show="form.$submitted || form.pass.$touched">
+                              <span ng-show="form.pass.$error.required">Enter your new password.</span>
+                              <span ng-show="form.pass.$error.pattern">Password must have 8 character, 1 letter and 1 number.</span>
+                           </div> -->
+                        </div>
+                        <div class="form-group">
+                           <label class="sr-only" for="confirmPassword">Confirm Password</label>
+                           <input type="password" class="form-control" id="confirmPassword" placeholder="Confirm Password" ng-model="usr_confirmPassword" >
```

Comment:
I think on smaller screen the 'password' and 'confirm password' inputs should be on separate lines